[22:43:44] {C:\Netwroking_Project\NetworkSecurity\exceptions\exception.py:17} INFO - Enter the try block
